deps.Dockerfile

```diff
@@ -11,11 +11,13 @@ RUN ["go", "build", "-ldflags", "-s -w", "."]
 RUN ["upx", "entrypoint"]
 
 
-FROM alpine as empty
+FROM alpine as base
 RUN ["mkdir", "/empty"]
+COPY rootfs/ /rootfs
```

Comment:
Why the trailing / ?

Reply:
Because it already was there in the old command copying it. To keep it consistent.

Reply:
It had the reason to copy rootfs/ to /, not to /rootfs/. Your case differs.

Reply:
I don't quite get this. Why do you differentiate if the source should have a trailing slash or not depending on whether the destination it `/`? Would `COPY rootfs/ /rootfs/` be fine?

Reply:
1. Because I consider the trailing / an exception, not the rule.
2. No, see 1.

Reply:
But shouldn't it then be `COPY rootfs /`? I consider it as a hint to show that we are actually copying a directory and not a file.

Reply:
If it works..